bin/target_user.sh

```diff
@@ -8,29 +8,33 @@
 #
 log_src='['${0##*/}']'
 
+odoo_user='odoo'
+
 # Check if there's a target user to run Odoo
 if [ "$TARGET_ID" ]; then
-    echo $log_src[`date +%F.%H:%M:%S`]' TARGET_ID = '$TARGET_ID
-
     echo $log_src[`date +%F.%H:%M:%S`]' Check if target user exists...'
-    EXISTS=$( getent passwd $TARGET_ID | wc -l )
-    echo $log_src[`date +%F.%H:%M:%S`]' EXISTS = '$EXISTS
+    exists=$( getent passwd $TARGET_ID | wc -l )
 
     # Create target user
-    if [ $EXISTS == "0" ]; then
-        echo $log_src[`date +%F.%H:%M:%S`]' Create target user...'
-        adduser --uid $TARGET_ID --disabled-password --gecos "" --shell=/bin/bash target-user
+    if [ $exists == "0" ]; then
+        echo $log_src[`date +%F.%H:%M:%S`]' Create target Odoo user...'
+        odoo_user='target-odoo-user'
+        adduser --uid $TARGET_ID --disabled-password --gecos "" --shell=/bin/bash $odoo_user
```

Comment:
The folder where the sources are is the home folder of `odoo` user. `target-odoo-user` doesn't need to have a home folder of his own, he simply needs to have access to the home folder of `odoo` (this is why we put him in the same group). As for the shell, it's quite convenient when logging inside the container.